torch/csrc/jit/compiler/include/schedule.h

```diff
@@ -551,14 +551,24 @@ Object* CloneObject(Object* object) {
 class Schedule : RefHandle<ScheduleNode> {
  public:
   static Schedule make(const std::vector<Tensor>& funcs) {
-    return Schedule(new ScheduleNode(funcs));
+    return std::move(Schedule(new ScheduleNode(funcs)));
```

Comment:
Is `std::move` actually necessary here?

Reply:
Good catch. It is a debug experiment that was supposed to be removed. Must have slipped my mind. :)

Reply:
Done in https://github.com/mruberry/pytorch/pull/35